slick/src/test/java/akka/stream/alpakka/slick/javadsl/DocSnippetFlowWithPassThrough.java

```diff
@@ -72,6 +75,7 @@ public static void main(String[] args) throws Exception {
         .via(
           Slick.flowWithPassThrough(
             session,
+            executorService,
```

Comment:
Just use the dispatcher one instead here. No reason to introduce an additional, manually lifecycle, thread pool here when the stream is tied to the actor system lifecycle